### DISCUSSION TITLE
Possible Typo?

### DISCUSSION THREAD

**Discussion by seb-afk** [dagster-io/dagster#6768] [Q&A]
Should this query-string not be: 
`?q[]=status%3AQUEUED` instead of 
`?q=status%3AQUEUED` ?

https://github.com/dagster-io/dagster/blob/e2b14d2e8b478c905431dcff5b1745ee221b260a/js_modules/dagit/packages/core/src/gantt/GanttChart.tsx#L811

**Comment by bengotow** [comment]
Thanks for filing this! I think you're correct, going to convert this into a github issue and we'll get it fixed 👍

**Reply by seb-afk** [reply]
Ok, great! I think it may apply here as well: https://github.com/dagster-io/dagster/blob/b3a870c25af3af67f06de6f00d4e09c8be6a767a/js_modules/dagit/packages/ui/src/components/NonIdealState.stories.tsx#L20
